components/pages/PlantProfile.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { View, Text } from 'react-native'
+
+import styles from '../../styles/styles';
+
+export default function PlantProfile() {
+  return (
+    <View>
+      <Text>PlantProfile Component</Text>
+    </View>
+  )
```

Comment:
When I emulate this on my IOS device, The text is hidden by the phones status bar. I know styling and adjustments will be made tomorrow. Let's keep in mind to wrap this in a SafeAreaView component to prevent the overlap. 🤓